packages/app-core/src/ui/App/ViewContainer.tsx

```diff
@@ -2,18 +2,24 @@ import React, { useEffect } from 'react'
 import { Paper, useTheme } from '@mui/material'
 import { makeStyles } from 'tss-react/mui'
 import { observer } from 'mobx-react'
-import { getSession, useWidthSetter } from '@jbrowse/core/util'
-import { IBaseViewModel } from '@jbrowse/core/pluggableElementTypes/models'
-import { SessionWithFocusedViewAndDrawerWidgets } from '@jbrowse/core/util'
+import {
+  AbstractViewModel,
+  SessionWithFocusedViewAndDrawerWidgets,
+  useWidthSetter,
+} from '@jbrowse/core/util'
 
 // locals
 import ViewHeader from './ViewHeader'
+import ViewWrapper from './ViewWrapper'
 
 const useStyles = makeStyles()(theme => ({
   viewContainer: {
-    overflow: 'hidden',
     margin: theme.spacing(0.5),
     padding: `0 ${theme.spacing(1)} ${theme.spacing(1)}`,
+    overflow: 'visible',
```

Comment:
are these changes to overflow visible necessary in all cases? i tried removing these two for example and it appeared to "still work" so just curious. I find myself often wanting to "minimize the amount of css" so if there are code comments explaining if/why something essential it would be good